homeassistant/components/verisure/__init__.py

```diff
@@ -145,3 +190,26 @@ async def enable_autolock(service):
         DOMAIN, SERVICE_ENABLE_AUTOLOCK, enable_autolock, schema=DEVICE_SERIAL_SCHEMA
     )
     return True
+
+
+async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Unload Verisure config entry."""
+    unload_ok = all(
+        await asyncio.gather(
+            *(
+                hass.config_entries.async_forward_entry_unload(entry, platform)
+                for platform in PLATFORMS
+            )
+        )
+    )
+
+    if unload_ok:
```

Comment:
I would turn this if-statement around. Now your `hass.data` check is also executed if unload was not ok. 
```python
    if not unload_ok:
        return False
```